documentation/asciidoc/topics/proc_enabling_client_cert_authn.adoc

```diff
@@ -0,0 +1,36 @@
+[id='enabling-client-certificate-authentication_{context}']
+= Enabling client certificate authentication
+
+[role="_abstract"]
+{brandname} clusters do not use client certificate authentication by default.
+Complete the following procedure so that clients must present valid certificates before they can connect to {brandname} endpoints.
```

Comment:
"must present valid certificates before they can connect to {brandname} endpoints." etc has already been said, maybe just:

"{brandname} clusters do not use client certificate authentication by default. Complete the following procedure to enable it."